index.html

```diff
@@ -75,13 +97,25 @@ <h2>Possibility for renewable electricity autarky in Europe</h2>
                     <li>Autarky is impossible (dark red), when demand exceeds technical potential.</li>
                 </ul>
             </p>
+            <h2>How do I use the map?</h2>
             <p>
-                The data is taken from the peer-reviewed article:
-                <q>Tröndle, T., Pfenninger, S., Lilliestam, J., 2019. Home-made or imported: on the
-                possibility for renewable electricity autarky on all scales in Europe. Energy
-                Strategy Reviews 26.
-                <a href=https://doi.org/10.1016/j.esr.2019.100388>https://doi.org/10.1016/j.esr.2019.100388</a></q>.
-                <b>If you make use of the data in academic work, please cite the article.</b>
+                You can zoom in and out to find out about the potential of renewable electricity
+                generation and whether the region can supply itself with it.
+                Depending on the region you look at, you will notice that the colour of the area may
+                change: from dark green - areas in which self-sufficiency is likely - to dark red
+                - areas in which self-sufficiency is not possible. If you hoover over the area of ​​your
+                choice, you will receive further information in a white field at the top right: You will
+                be shown whether electrical self-sufficiency is possible, for example, in the state of
+                Brandenburg in Germany, where our institute is based. You can also see how many people
+                live there, how much electricity they need, and the potential to produce electricity from the sun and wind. Have fun trying it out!
+            </p>
+            <h2>Where do I get more information about the project?</h2>
+            <p>
+                In our scientific article you will find more background information on the data and the
+                method: <a href=https://doi.org/10.1016/j.esr.2019.100388>Tröndle, T., Pfenninger, S., Lilliestam, J., 2019. Home-made or imported: on the possibility for renewable electricity autarky on all scales in Europe. Energy Strategy Reviews 26.</a>. If you have the scientific data in the article, please quote it.
```

Comment:
`If you have the scientific data in the article, please quote it` 

Doesn't make sense. I think the previous sentence `If you make use of the data in academic work, please cite the article.` was fine.

Reply:
you are right. changed